### ISSUE TITLE
Browser-global extended distribution variant missing

### ISSUE THREAD

**Issue by j-hellenberg** [github/webauthn-json#61]
Hi all,

during a recent project we wanted to use the `browser-global` dist version but noticed it is only exported in the basic and not the extended variant. Therefore, we'd like to ask whether you would be interested in adding that version to the `script/build-js.js`. As we've already done that in our fork to get our project to work, I've also just submitted a small PR (#60) for that in case you want to take a look :slightly_smiling_face:

**Comment by lgarron** [comment]
Hi! Thanks for offering the contribution!

I'd really like to keep the scope of this project as small as possible, which includes avoiding new builds if possible.

Speaking as the primary maintainer, I'm reluctant to offer the extended build a `browser-global` version because:

- `browser-global` builds are an anti-pattern now than all browsers have supported modules for quite a while — I don't think I'd even include one I were publishing this library from scratch today, since the ESM builds can be used directly (even for setting browser globals to be used in non-module scripts).
- It's more likely to be fragile and break in subtle ways due to browser or spec changes.
- We never ended up using the extended build at GitHub ourselves — it's still maintained, but kind of provided "as-is" and I'd like to discourage usage unless necessary.
- It's much easier to get ecosystem insight about projects using `npm` for package dependencies (e.g. so they show up [here](https://github.com/github/webauthn-json/network/dependents?package_id=UGFja2FnZS01MDAxODgzOTM%3D)), so I'd like to steer people towards using that if possible.
- If you really need to use the extensions, you should probably "know what you're doing", in which case you should be able to use the build from `npm` or make your own build if you really need (as in your case).

I hope it's okay if I decline the PR for now, but I'm happy to reconsider if we get more requests for this.

To that end:

- I'll leave this issue open for now.
- Could you share why you are motivated to use a browser-global extended build?

**Comment by j-hellenberg** [comment]
I think that's some pretty good reasoning you came up with there.

Regarding your question about why we wanted to do this, I must admit it was mostly about convenience. We were doing a research project and used the library for our proof-of-concept implementation which required some of the extensions, mainly the `largeBlob` extension (the [profile page of the organisation we created for the project](https://github.com/All-Your-Locks-Are-Belong-To-Us) provides a very brief summary on the topic in case you're interested). Therefore, while we could probably have made it work using the module build with a bit more effort (or maybe a bit more experience on using modules in a non-module context, for that matter), we decided to go with the just-plug-it-into-a-script-tag-and-use-it-everywhere solution for our prototype.

Obviously, if we wanted to turn that prototype into a more fully-fledged solution, we'd put more attention to actually following conventions. Therefore, we're fully fine with you declining the proposal. We just wanted to bring it up in case it was missing due to an oversight instead of a conscious decision :slightly_smiling_face:
